### ISSUE TITLE
Interoperability with nb

### ISSUE THREAD

**Issue by chmac** [barfuin/pagemarks#2]
I've been looking at [nb](https://github.com/xwmx/nb) as another tool that stores bookmarks. It uses git+markdown. The author has recently added some [specs](https://github.com/xwmx/nb#specifications) on the file / repo format. I wonder if it would be possible for pagemarks to support the same format...

I guess that the default format here is [this yaml file](https://github.com/barfuin/pagemarks/blob/master/_data/bookmarks.yml) like so:

https://github.com/barfuin/pagemarks/blob/dcc0e7296839768a3a0a5bd5b420d3edc9f49899/_data/bookmarks.yml#L9-L16

I don't know Jekyll so well, but I have some experience with Gatsby. In Gatsby it would be possible to write some kind of transformation plugin to suck content in from another source, and generate something in the yaml format you're using here.

Food for thought...

**Comment by barfuin** [comment]
Right, an nb importer would be rather easy, as all you'd need to do is convert their format into our yaml like you showed.

I didn't know of nb until you mentioned it. It looks much larger, and somehow more complicated, although also quite complete. We'll have some way to go until we reach the same level of completeness, but then my guess is that pagemarks will be easier to use and more lightweight.
